security/pfSense-pkg-suricata/files/usr/local/pkg/suricata/suricata_migrate_config.php

```diff
@@ -129,12 +117,12 @@
 /* from the package configuration. The status is now      */
 /* stored in a local file.                                */
 /**********************************************************/
-if (isset($config['installedpackages']['suricata']['config'][0]['last_rule_upd_status'])) {
-	unset($config['installedpackages']['suricata']['config'][0]['last_rule_upd_status']);
+if (config_path_enabled('installedpackages/suricata/config/0', 'last_rule_upd_status')) {
```

Comment:
For these and the following, I'd suggest straying away from config_path_enabled() as the intent there is to check if a particular element or property is enabled, with the ultimate goal of being able to abstract out what the implementation of 'enabled' is in the backend of the config. I'd recommend instead using empty as above, or if these properties are optional use `config_get_path('foo/bar/baz') !== null`